src/pages/api/configs/tryToggleEdit.tsx

```diff
@@ -0,0 +1,33 @@
+import Consola from 'consola';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+function Post(req: NextApiRequest, res: NextApiResponse) {
+  const { tried } = req.body;
+  // Try to match the password with the EDIT_PASSWORD env variable
+  if (tried === process.env.EDIT_MODE_PASSWORD) {
+    process.env.DISABLE_EDIT_MODE = process.env.DISABLE_EDIT_MODE === 'true' ? 'false' : 'true';
```

Comment:
Seems inefficient. Consider negation

Reply:
What do you mean negation ? If the password is wrong the function will continue and warn

Reply:
Something along the lines of
```
process.env.DISABLE_EDIT_MODE = !(process.env.DISABLE_EDIT_MODE === 'true')
```